doc/manual/variables-and-scoping.rst

```diff
@@ -79,23 +83,36 @@ Modules can introduce variables of other modules into their scope
 through the :ref:`using or import <man-modules>` statements or through
 qualified access using the dot-notation, i.e. each module is a
 so-called *namespace*.  Note that variable bindings can only be
-changed within their global scope and not from an outside module. ::
-
-    module A
-    a = 1 # a global in A's scope
-    end
-
-    module B
-    # b = a # would error as B's global scope is separate from A's
-        module C
-        c = 2
-        end
-    b = C.c # can access the namespace of a nested global scope
-            # through a qualified access
-    import A # makes module A available
-    d = A.a
-    # A.a = 2 # would error with: "ERROR: cannot assign variables in other modules"
-    end
+changed within their global scope and not from an outside module. :
```

Comment:
If the `::` is handled by the `doctest` directive, doesn't this result in a stray colon at end of line?